src/main/steps/sequence.ts

```diff
@@ -0,0 +1,172 @@
+import { Request } from 'express';
+
+import {
+  HAS_RELATIONSHIP_BROKEN_URL,
+  PARTNER_GENDER_URL,
+  RELATIONSHIP_NOT_BROKEN_URL,
+  UNION_CERTIFICATE_URL,
+} from './urls';
+
+export interface Step {
+  id: string;
+  title: string;
+  field: string;
+  url: string;
+  subSteps?: SubStep[];
+}
+
+interface SubStep extends Step {
+  when: (response: Record<string, string>) => boolean;
+  finalPage?: boolean;
+}
+interface IncompleteStep {
+  id?: string;
+  title: string;
+  field?: string;
+  url?: string;
```

Comment:
Why do some steps have a url and others don't?

Reply:
This is a temporary type, once we've added all the steps and substeps this can go away. There's a method below that gets all the valid steps (which should contain an ID and URL). I've added a `@TODO` to remind us to remove it in 9b52937.